README.md

```diff
@@ -50,22 +50,41 @@ Responsibilities and roles of the bridge:
 
 ## Usage
 
-### Install Dependencies
+There are two ways to deploy contracts: one option is to install and use NodeJS directly on the system,
+another way is for those systems where it is more preferable to use NodeJS in the docker environment. 
+
+### Deployment with NodeJS
+
+#### Install Dependencies
 ```bash
 npm install
 ```
-### Deploy
+#### Deploy
 Please the [README.md](deploy/README.md) in the `deploy` folder for instructions and .env file configuration
 
-### Test
+#### Test
 ```bash
 npm test
 ```
 
-### Flatten
+#### Flatten
+The flatten contracts can be used to verify code of contracts in a block explorer like BlockScout or Etherscan.
```

Comment:
~~ The flatten contracts ~~ Flattened contracts

Reply:
Are any additional instructions needed to verify using flattened contracts, or is this self-explanatory?

Reply:
Statement fixed.
These steps are specific for the particular block explorer. Don't think that necessary to cover here how to verify contracts in the explorer.